rules-tests/CodeQuality/Rector/Property/TypedPropertyFromToOneRelationTypeRector/Fixture/doctrine_attribute_many_to_one.php.inc

```diff
@@ -33,7 +33,7 @@ class DoctrineManyToOne
 
     #[ORM\ManyToOne(targetEntity:"Rector\Doctrine\Tests\CodeQuality\Rector\Property\TypedPropertyFromToOneRelationTypeRector\Source\Company", inversedBy:"userBuildings")]
     #[ORM\JoinColumn(name:"company_id", referencedColumnName:"id", nullable:false)]
-    private ?\Rector\Doctrine\Tests\CodeQuality\Rector\Property\TypedPropertyFromToOneRelationTypeRector\Source\Company $company3 = null;
+    private \Rector\Doctrine\Tests\CodeQuality\Rector\Property\TypedPropertyFromToOneRelationTypeRector\Source\Company $company3;
```

Comment:
nullable: false -> so this property should not be null and we don't need a new Testcase. Both changes are already covered